### PR TITLE
cradle*: add German translation

### PR DESCRIPTION
**Pull request by flyck**
- [X] The page (if new), does not already exist in the repo.
- [X]  The page is in the correct platform directory (common/, linux/, etc.)
- [X]  The page has 8 or fewer examples.
- [X]  The PR title conforms to the recommended templates.
- [X]  The page follows the content guidelines.
- [X]  The page description includes a link to documentation or a homepage (if applicable).

Hello!

I translated the cradle pages into German.

### CODE REVIEW

File: pages.de/common/cradle-elastic.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Management der ElasticSearch Instanzen einer Cradle Instanz.
+> Weitere Informationen: <https://cradlephp.github.io/docs/3.B.-Reference-Command-Line-Tools.html#elastic>.
+
+- Entleeren des ElasticSearch Indexes:
+
+`cradle elastic flush`
+
+- Entleeren des ElasticSearch Indexes für ein bestimmtes Paket:
+
+`cradle elastic flush {{package_name}}`
```
Comment (navarroaxel) [inline pages.de/common/cradle-elastic.md:12]: Please translate the arguments for get this easier for non-English speakers in all pages.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @flyck, I left 1 comment.

File: pages.de/common/cradle-elastic.md (hunk 1 above)
Comment (flyck) [reply pages.de/common/cradle-elastic.md]: Good catch, of course 👍

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**